content/en/docs/concepts/overview/components.md

```diff
@@ -50,26 +50,28 @@ the same machine, and do not run user containers on this machine. See
 
 These controllers include:
 
-  * Node Controller: Responsible for noticing and responding when nodes go down.
-  * Replication Controller: Responsible for maintaining the correct number of pods for every replication
+  * Node controller: Responsible for noticing and responding when nodes go down.
+  * Replication controller: Responsible for maintaining the correct number of pods for every replication
   controller object in the system.
-  * Endpoints Controller: Populates the Endpoints object (that is, joins Services & Pods).
-  * Service Account & Token Controllers: Create default accounts and API access tokens for new namespaces.
+  * Endpoints controller: Populates the Endpoints object (that is, joins Services & Pods).
+  * Service Account & Token controllers: Create default accounts and API access tokens for new namespaces.
 
 ### cloud-controller-manager
 
-[cloud-controller-manager](/docs/tasks/administer-cluster/running-cloud-controller/) runs controllers that interact with the underlying cloud providers. The cloud-controller-manager binary is an alpha feature introduced in Kubernetes release 1.6.
+{{< glossary_definition term_id="cloud-controller-manager" length="short" >}}
 
-cloud-controller-manager runs cloud-provider-specific controller loops only. You must disable these controller loops in the kube-controller-manager. You can disable the controller loops by setting the `--cloud-provider` flag to `external` when starting the kube-controller-manager.
+The cloud-controller-manager only runs controllers that are specific to your cloud provider.
+If you are running Kubernetes on your own premises, or in a learning environment inside your
+own PC, the cluster does not have a cloud controller manager.
```

Comment:
nit: Are lines 64-65 needed?

Reply:
The learning outcome here is: the reader understands that they can run Kubernetes without deploying it to a public cloud.